djstripe/middleware.py

```diff
@@ -50,12 +61,13 @@ class SubscriptionPaymentMiddleware(object):
     """
 
     def process_request(self, request):
+        """Check the subscriber's subscription status.
 
-        # Does the request match any of the docstring rules?
+        Returns early if request doesnt match rules in docstring.
```

Comment:
Returns early if the request does not match the rules outlined in this middleware's docstring.